### ISSUE TITLE
Error in ncurses UI terminal  size

### ISSUE THREAD

**Issue by lordmorgul** [lordmorgul/ham2mon#15]
A non-descript error occurs if the terminal running ham2mon is too short or too narrow.  If the program is already running and the terminal is resized too small in either dimension the same issue but varied error is displayed.
Todo explore if try/catch can improve error output.

**Comment by john-** [comment]
My initial attempt at this got sidelined by just making the RECEIVER section [2 columns](https://github.com/john-/ham2mon/blob/master/ham2mon.png).  This change got the minimum size small enough that I gave up on addressing the root issue you raised.

Related to this my fork moved the top level of ham2mon to asyncio.  I am not sure if this was the right thing to do.  However, it seemed to let me make dynamic resizing more robust and fix some of the glitches that occurred before I moved to asyncio.
